libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -78,10 +133,14 @@ internal class RouteRefreshController(
         try {
             repeat(FAILED_ATTEMPTS_TO_INVALIDATE_EXPIRING_DATA) {
                 timeUntilNextAttempt.await()
+                if (it == 0) {
```

Comment:
nit: This shouldn't be needed since we only notify on distinct state changes. In any case, we can leave as is.

Reply:
this is kinda doc, that `start` is triggered once per retry